test/suite/wtbackup.py

```diff
@@ -343,12 +343,14 @@ def take_log_backup(self, bkup_c, backup_dir, orig_logs, log_cursor=None):
     # Optional arguments:
     #   consolidate: Add consolidate option to the cursor.
```

Comment:
You may want to update the header describing the new args

Reply:
[6933a2e](https://github.com/wiredtiger/wiredtiger/pull/9977/commits/6933a2ea61497bafac665a9f342e788504d8b62f)